### PR TITLE
feat(frontend): add Authenticate icon button in header when signed out [GIX-3044]

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We include the authenticate button in the header when signed out.

Screenshot 2024-10-02 at 16 33 35
Screenshot 2024-10-02 at 16 33 43
Screenshot 2024-10-02 at 16 34 45
Screenshot 2024-10-02 at 16 34 53

### CODE REVIEW

File: src/frontend/src/lib/components/icons/IconLogin.svelte [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!-- source: ISC Lucide - please visit https://lucide.dev/license -->
```
[the remark this answers is not in the document]
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/icons/IconLogin.svelte:1]: It is a new library that the designer team would like to use. According to the [official site](https://lucide.dev/license):

> Permission to use, copy, modify, and/or distribute this software for any purpose with or without fee is hereby granted, provided that the above copyright notice and this permission notice appear in all copies.

So, I put it there. Honestly I am not really sure if I should put `Lucide License` or `ISC License`

[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/lib/components/icons/IconLogin.svelte:1]: Sounds good. I'll approve the PR and ask a review internally.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Comment by AntonioVentilii-DFINITY** [comment]
@peterpeterparker I installed the icon library (since we discussed it offline). Please re-review

**Pull request merged**